src/meili/traffic_segment_matcher.cc

```diff
@@ -42,6 +42,8 @@ namespace {
         return true;
       }
     }
+    //not connected
+    return false;
```

Comment:
so yeah in c99 and up you can have functions without returns and without extra flags the compiler will say nothing. forgetting to put this return here made sections of traces that had discontinuities seem like they were continuous, this would have the effect of messing up the reported lengths